site/content/quickstart/binaries.md

```diff
@@ -0,0 +1,97 @@
+---
+linktitle: "Binaries"
+weight: 3
+---
+
+<!-- TODO: Combine with other quickstart? -->
```

Comment:
Seems like we have two copies of this and the kube guide?

Reply:
OK, the K8s one was duplicated…The other guides are slightly different from each other… 

Anything in the quickstart section is single node only (which is why there's no K8s guide, makes no sense single node), anything in the "Create a Cluster" section is for creating a cluster. There is some similarity, but should be fairly different.